test/vanilla-eap/test-app-local-provisioning/galleon/provisioning.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" ?>
 
 <installation xmlns="urn:jboss:galleon:provisioning:3.0">
-    <feature-pack location="org.jboss.eap:wildfly-ee-galleon-pack:8.0.0.Beta-redhat-20220408">
+    <feature-pack location="org.jboss.eap:wildfly-ee-galleon-pack:">
```

Comment:
is that correct or should the trailing `:` be removed?

Reply:
Yes, we are not handling the provisioning.xml content to deal wit hthe trailing ":", it is needed.